### PR TITLE
Language fixes

### PR DESCRIPTION
**Pull request by Gwenwyfar**

### CODE REVIEW

File: Themes/default/languages/Search.english.php [hunk 1]
```diff
@@ -108,7 +108,7 @@
 
 $txt['search_engines_description'] = 'From this area you can decide in what detail you wish to track search engines as they index your forum as well as review search engine logs.';
 $txt['spider_mode'] = 'Search Engine Tracking Level';
-$txt['spider_mode_note'] = 'Note that logging of every spider action only occurs if tracking is set to either "high" or "very high". Detail of every spiders action is only logged if tracking is set to "very high".';
+$txt['spider_mode_note'] = 'Note that logging of every spider action only occurs if tracking is set to either "moderate" or "agressive". Detail of every spider\'s action is only logged if tracking is set to "agressive".';
```
Comment (Sesquipedalian) [inline Themes/default/languages/Search.english.php]: Spelling: "aggressive"

**Review by Sesquipedalian — CHANGES_REQUESTED** [review]
Small spelling error

**Comment by illori** [comment]
if you want to fix another one...
`Your request has been submitted, please be patient while the request is considered..`

we dont need 2 periods at the end of this one

**Comment by illori** [comment]
maybe one more?
`Thank you for submitting this report. You should hear back from the admin team shortly.`

IMO should be administration and leave out team completely.

**Comment by Gwenwyfar** [comment]
Done. The first one had already been fixed, @illori.

**Comment by Gwenwyfar** [comment]
Oh, the first one is an extra . in the template.

**Review by Sesquipedalian — APPROVED** [review]

**Pull request merged**